lib/OpenQA/Schema/ResultSet/Assets.pm

```diff
@@ -95,7 +95,7 @@ sub refresh_assets {
     my ($self) = @_;
 
     while (my $asset = $self->next) {
-        my $is_fixed = $asset->is_fixed;
+        my $is_fixed = $asset->is_fixed ? 1 : 0;
```

Comment:
Maybe this should be changed in the `sub is_fixed` directly?

Reply:
It should only be required here but I can also change `is_fixed` itself.